vendor/vendor.json

```diff
@@ -75,17 +75,9 @@
 			"versionExact": "v17.4.0"
 		},
 		{
-			"checksumSHA1": "1+iufNHv/JIZ3efJVyy8R8ByEro=",
-			"path": "github.com/Azure/azure-sdk-for-go/services/dns/mgmt/2016-04-01/dns",
-			"revision": "7971189ecf5a584b9211f2527737f94bb979644e",
-			"revisionTime": "2018-06-19T23:05:49Z",
-			"version": "=v17.4.0",
-			"versionExact": "v17.4.0"
-		},
-		{
-			"checksumSHA1": "yPJb1PNC7uqpUi6GtgfD4wt/Xss=",
-			"path": "github.com/Azure/azure-sdk-for-go/services/dns/mgmt/2017-10-01/dns",
-			"revision": "7971189ecf5a584b9211f2527737f94bb979644e",
+			"checksumSHA1": "t6ChlDNU9eaDOo9OgEf/RpWbLZE=",
+			"path": "github.com/Azure/azure-sdk-for-go/services/dns/mgmt/2018-03-01-preview/dns",
```

Comment:
this SDK's actually deprecated - could we switch this out for `github.com/Azure/azure-sdk-for-go/services/preview/dns/mgmt/2018-03-01-preview/dns` which is the same but doesn't have the deprecation

Reply:
done.